components/editor/commands/insertSound.js

```diff
@@ -0,0 +1,21 @@
+import { updateMark } from 'tiptap-commands'
+
+export default function(type, attrs, text = 'SOUND') {
```

Comment:
The default should probably also follow a scheme like ` [SOUND] `.

Reply:
See line https://github.com/tactilenews/spacemuffin/pull/80/files#diff-4544ea27877906c7fac041260d7f8d7aR40